lib/rubocop/cop/style/space_after_not.rb

```diff
@@ -16,9 +16,9 @@ class SpaceAfterNot < Cop
         MSG = 'Do not leave space between `!` and its argument.'.freeze
 
         def on_send(node)
-          receiver, method_name, *_args = *node
+          receiver, _method_name, *_args = *node
 
-          return unless method_name == :!
+          return unless node.keyword_bang?
```

Comment:
Yet this will still unpack the node twice, right? If we move this guard to the first line, can we save the double unpack for most of the cases where we'll not hit a bang keyword?